### PR TITLE
Added navigational link for ephemeral and persistent volume

### PR DESCRIPTION
**Pull request by niranjandarshann**
Fixes Adding navigational link to the [Ephemeral volume] and [persistent volume] #40778

### CODE REVIEW

File: content/en/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ drivers, but the functionality is somewhat limited.
 
 Kubernetes supports many types of volumes. A {{< glossary_tooltip term_id="pod" text="Pod" >}}
 can use any number of volume types simultaneously.
-Ephemeral volume types have a lifetime of a pod, but persistent volumes exist beyond
+[Ephemeral volume](/docs/concepts/storage/ephemeral-volumes/) types have a lifetime of a pod, but [persistent volumes](/docs/concepts/storage/persistent-volumes/) exist beyond
```
Comment (tengqm) [inline content/en/docs/concepts/storage/volumes.md:35]:
```suggestion
[Ephemeral volume](/docs/concepts/storage/ephemeral-volumes/) types have a lifetime of a pod,
but [persistent volumes](/docs/concepts/storage/persistent-volumes/) exist beyond
```

**Comment by niranjandarshann** [comment]
Thank you @tengqm  for the suggestion commited those changes. Please take a look

**Review by dipesh-rawat — COMMENTED** [review]
The changes looks good. But I see the CLA is failing after the latest commit. @niranjandarshann Could you sort out the CLA signing after which this can be reviewed and merged.

/easycla

**Comment by niranjandarshann** [comment]
> The changes looks good. But I see the CLA is failing after the latest commit. @niranjandarshann Could you sort out the CLA signing after which this can be reviewed and merged.
> 
> /easycla

@dipesh-rawat  now the EasyCLA test passes. Requesting you for review and merging

**Comment by niranjandarshann** [comment]
> The changes looks good. But I see the CLA is failing after the latest commit. @niranjandarshann Could you sort out the CLA signing after which this can be reviewed and merged.
> 
> /easycla

@dipesh-rawat  pinging for review and merging

**Review by dipesh-rawat — COMMENTED** [review]
Addition of the links to the volume pages works as expected.
/lgtm

Deploy Preview: [deploy-preview-40785-storage/volumes](https://deploy-preview-40785--kubernetes-io-main-staging.netlify.app/docs/concepts/storage/volumes/)

**Comment by niranjandarshann** [comment]
pinging @annajung for approval.

**Pull request merged**